src/components/templates/Playa/Playa.tsx

```diff
@@ -527,26 +529,23 @@ const Playa = () => {
     });
   }, [myXRef, myYRef]);
 
-  const getNearbyVenue = useCallback(
-    (x: number, y: number) => {
-      if (!venues) return;
-      let closestVenue: WithId<AnyVenue> | undefined;
-      let distanceToClosestVenue: number;
-      venues.forEach((venue) => {
-        const distance = distanceToVenue(x, y, venue.placement);
-        if (
-          distance &&
-          distance <= VENUE_NEARBY_DISTANCE &&
-          (!distanceToClosestVenue || distance < distanceToClosestVenue)
-        ) {
-          closestVenue = venue;
-          distanceToClosestVenue = distance;
-        }
-      });
-      return closestVenue;
-    },
-    [venues]
-  );
+  const getNearbyVenue = useCallback((x: number, y: number) => {
+    if (!venues) return;
```

Comment:
```suggestion
    if (!venues) return;
    
```

Reply:
@goran-peoski-work if you think something is irrelevant, please say it out loud. I've just noticed that it's the outdated Playa template, so this comment is irrelevant, but you shouldn't resolve the comments silently